specification/devcenter/DevCenter/Environments/routes.tsp

```diff
@@ -84,6 +84,41 @@ interface Environments {
     }
   >;
 
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Standard operations produce wrong swagger"
+  @doc("Partially updates an environment.")
+  @route("/projects/{projectName}/users/{userId}/environments/{environmentName}")
+  @patch
+  @added(APIVersions.v2024_02_01)
+  patchEnvironment is Azure.Core.Foundations.Operation<
+    {
+      @doc("Set consumer to JSON merge patch.")
+      @header
+      contentType: "application/merge-patch+json";
+
+      @doc("The DevCenter Project upon which to execute operations.")
+      @path
+      projectName: string;
+
+      @doc("The AAD object id of the user. If value is 'me', the identity is taken from the authentication context.")
+      @path
+      userId: string;
+
+      @doc("The name of the environment.")
+      @path
+      environmentName: string;
+
+      @doc("Represents an environment.")
+      @body
+      body: Environment;
+    },
+    {
+      @statusCode
+      statusCode: 200;
+
+      @body body: Environment;
+    }
```

Comment:
I believe if you just put Environment here, the default status code is already 200 so no need to expand here:
```suggestion
    Environment
```